exercises/gigasecond/src/example/java/Gigasecond.java

```diff
@@ -1,19 +1,20 @@
 import java.time.LocalDateTime;
 import java.time.LocalDate;
 
-public class Gigasecond {
+class Gigasecond {
 
-    private LocalDateTime birthDateTime;
+    private LocalDateTime initialDateTime;
```

Comment:
I feel like we've had a very similar discussion before, so apologies, but: I was referring to the [canonical description](https://github.com/exercism/x-common/blob/master/exercises/gigasecond/description.md) for this problem and it doesn't mention birthdates (which is why I made the adjustment I did). Please remind me where I can find the reference you're thinking of? Is the [blurb](https://github.com/exercism/x-common/blob/master/exercises/gigasecond/metadata.yml#L2) always combined with the `description.md` file? Thanks!